node/full_client_test.go

```diff
@@ -813,8 +813,8 @@ func indexBlocks(t *testing.T, rpc *FullClient, heights []int64) {
 						Type: "begin_event",
 						Attributes: []abci.EventAttribute{
 							{
-								Key:   []byte("proposer"),
-								Value: []byte("FCAA001"),
+								Key:   string([]byte("proposer")),
+								Value: string([]byte("FCAA001")),
```

Comment:
Keep it simple :wink: 
```suggestion
								Key:   "proposer",
								Value: "FCAA001",
```

Reply:
oops, apologies for this lol